src/main/java/org/broadinstitute/consent/http/models/InstitutionDomainMap.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.consent.http.models;
+
+import java.util.List;
+import java.util.Map;
+
+public class InstitutionDomainMap {
+
+  private Map<String, List<String>> institutionDomainMap;
```

Comment:
Do we expect that order matters in the List<>?

Reply:
I wasn't anticipating that it would.

Reply:
Would Set be more appropriate?

Reply:
I had a version with a Set, but ultimately didn't see a tremendous advantage for operations like `contains` since the lists are likely to be relatively short. I can switch back, however, if you think that would clarify functionality.

Reply:
`List` to me implies two things I want to pay attention to:

1. Order
2. Duplicates

It sounds like we won't have to worry about either.  IMHO, `Set` conveys that clearer than `List`.  That being said, what I've written in my own code as a placeholder is Map<String, Collection<String>> with the impl creating a HashSet<>().  I think that probably gives us the most flexibility here.

Reply:
Ok, I'll switch to `Set`